src/Core/Utility/PackageHelper.cs

```diff
@@ -15,6 +15,15 @@ public static bool IsManifest(string path)
             return Path.GetExtension(path).Equals(Constants.ManifestExtension, StringComparison.OrdinalIgnoreCase);
         }
 
+        public static bool IsPackageManifest(string path, string packageId)
+        {
+            return path.EndsWith(packageId + Constants.ManifestExtension, StringComparison.OrdinalIgnoreCase);

```

Comment:
this should use Path.GetFileName instead of ends with since it could get things like aaaaPackageId.nuspec